test/Twig/Tests/ErrorTest.php

```diff
@@ -54,6 +65,37 @@ public function testTwigExceptionAddsFileAndLineWhenMissingWithInheritanceOnDisk
         }
     }
 
+    public function testTwigExceptionAddsFileAndLineWhenMissingWithInheritanceOnFilesystem()
+    {
+        $loader = new Twig_Loader_Filesystem(dirname(__FILE__).'/Fixtures/errors');
+        $twig = new Twig_Environment($loader, array('strict_variables' => true, 'debug' => true, 'cache' => false));
+
+        $template = $twig->loadTemplate('index.html');
+        try {
+            $template->render(array());
+
+            $this->fail();
+        } catch (Twig_Error_Runtime $e) {
+            $this->assertEquals('Variable "foo" does not exist.', $e->getMessage());
+            $this->assertEquals(3, $e->getTemplateLine());
+            $this->assertEquals('index.html', $e->getTemplateName());
+            $this->assertEquals(3, $e->getLine());
+            $this->assertEquals(strtr(dirname(__FILE__).'/Fixtures/errors/index.html', '/', DIRECTORY_SEPARATOR), $e->getFile());
+        }
+
+        try {
+            $template->render(array('foo' => new Twig_Tests_ErrorTest_Foo()));
+
+            $this->fail();
```

Comment:
please use separate tests instead of doing several tests in the same PHPUnit test. Ii would make things much easier to maintain (and would provide better reporting on failures, as your second test currently does not run at all if the first one fails)